packages/canary-web-components/src/components/ic-select-with-multi/ic-select.tsx

```diff
@@ -1140,11 +1140,11 @@ export class Select {
     return (
       <Host
         class={{
-          disabled: disabled,
-          searchable: searchable,
-          small: small,
-          [size]: size !== "default",
-          "full-width": fullWidth,
+          ["ic-select-disabled"]: disabled,
+          ["ic-select-searchable"]: searchable,
+          ["ic-select-small"]: small,
+          [`ic-select-${size}`]: size !== "default",
+          "ic-select-full-width": fullWidth,
```

Comment:
```suggestion
          ["ic-select-full-width"]: fullWidth,
```

just be good to standardize on use of square brackets (or not) for string values - just to match others on lines above